src/js/blue-button/containers/Main.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Breadcrumbs from '../components/Breadcrumbs';
+import ErrorableRadioButtons from '../../common/components/form-elements/ErrorableRadioButtons';
+import ErrorableCheckbox from '../../common/components/form-elements/ErrorableCheckbox';
+import DatePicker from 'react-datepicker';
+import { reportTypes } from '../config';
+
+class Main extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleStartDateChange = this.handleStartDateChange.bind(this);
+    this.handleEndDateChange = this.handleEndDateChange.bind(this);
+  }
+
+  handleStartDateChange() {
+
+  }
+
+  handleEndDateChange() {
+
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+
+    this.context.router.push('/download');
+  }
+
+  renderInformationTypes() {
+    return Object.keys(reportTypes).map(k => {
+      const rt = reportTypes[k];
+      return (
+        <div key={k} className="info-type-section">
+          <h5>{rt.title}</h5>
+          {rt.children.map(c => {
+            return (
+              <div key={c.value}>
+                <ErrorableCheckbox
+                    name={c.value}
+                    label={c.label}
+                    onValueChange={() => {}}/>
+              </div>
+            );
+          })}
+        </div>
+      );
+    });
+  }
+
+  render() {
+    // TODO: clean this up and hook up to action/reducer
+    const radioButtonProps = {
+      label: '',
+      options: [
+        { label: '3 months', value: '3mo' },
+        { label: '6 months', value: '6mo' },
+        { label: '1 year', value: '1yr' },
+        {
+          label: (
```

Comment:
I think this label is failing the propType validation on `ErrorableRadioButtons` because it's an `element` instead of `string`. I would make the propType a `oneOfType` with `string` and `element`.